src/catch2/catch_config.cpp

```diff
@@ -113,6 +113,25 @@ namespace Catch {
     bool Config::listReporters() const      { return m_data.listReporters; }
     bool Config::listListeners() const      { return m_data.listListeners; }
 
+    bool Config::hasBazel() const {
+#ifdef CATCH_CONFIG_BAZEL_SUPPORT
+        return true;
+#endif
+	
+#    if defined( _MSC_VER )
+        // On Windows getenv throws a warning as there is no input validation,
+        // since the switch is hardcoded, this should not be an issue.
+#        pragma warning( push )
+#        pragma warning( disable : 4996 )
+#    endif
+
+	return std::getenv("BAZEL_TEST") != nullptr;
```

Comment:
If you can, try to use clang-format, we provide a setting file in-repo. Otherwise, Catch2 doesn't use tabs, only spaces.